### ISSUE TITLE
Is PasteBar App an OPEN-source project？Where is the license

### ISSUE THREAD

**Issue by toyo2333** [PasteBar/PasteBarApp#138]
IS PasteBarAPP a OPEN-source project？where is the license

**Comment by kurdin** [comment]
@toyo2333 Yes it is, all source code is here in this repo, you can build and run it locally, you can use it for free for personal use but you can't resell it or create your own project from the source code and redistributed it under another name for commercial use. 

https://github.com/PasteBar/PasteBarApp/blob/main/CC-LICENSE

**Comment by TyMick** [comment]
Creative Commons [discourages use of their licenses for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software), recommending instead licenses that [the Free Software Foundation considers "free"](https://www.gnu.org/licenses/license-list#SoftwareLicenses) and [the Open Source Initiative considers "open source"](https://opensource.org/licenses). I think the GNU GPL would be a good fit, personally.

**Comment by maxwell-kalin** [comment]
Its source available and not open source
